src/components/slider/slider.api.json

```diff
@@ -85,13 +85,5 @@
     },
     {"name": "testID", "type": "string", "description": "The component test id"},
     {"name": "migrate", "type": "boolean", "description": "Temporary prop required for migration to the Slider's new implementation"}
-  ],
-  "snippet": [
-    "<Slider",
```

Comment:
Why removing the Slider snippet ?

Reply:
We could leave it but IMO we should not support components we want to deprecate (we have the `Incubator.Slider` that will be added in another PR).